app/controllers/documents_controller.rb

```diff
@@ -36,50 +35,34 @@ def archive
   end
 
   def new
-    @document_form = DocumentWizard::UploadForm.new
+    @document = @planning_application.documents.build
   end
 
   def create
-    @document_form = DocumentWizard::ConfirmUploadForm.new(
-      document_upload_confirmation_params,
-    )
+    @document = @planning_application.documents.build
+    unless document_params[:file].nil?
+      document_params.each do |_param|
+        @blob = ActiveStorage::Blob.find_signed(document_params[:file])
```

Comment:
Do we need to do this @blob and then .attach now? Can we not just pass the file param to the model and activestorage does magic?